jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/textfield/TypedTextField.java

```diff
@@ -176,6 +182,10 @@ public void setValue(String value) {
 
     protected void setValueInternal(@Nullable V modelValue, String presentationValue, boolean fromClient) {
         try {
+            if (isTrimEnabled()) {
```

Comment:
Move to `convertToModel`

Reply:
If I move it to `convertToModel`, this is what happens:
1. The value set from the server-side will not be trimmed:
   If you try:
   ```java
   textField.setTypedValue("   test    ");
   ```
   Then `modelValue` will not be `null`, so the conversion will not be processed.
   ![image](https://github.com/user-attachments/assets/7534fccb-768c-4743-8291-1e6a0d9d54f2)
2. `typedValue` will not be equal `value`:
   ![image](https://github.com/user-attachments/assets/057ba892-ea42-4b99-a19c-c158607d6b8b)
   ![image](https://github.com/user-attachments/assets/d624537c-6e1d-42f1-a93a-19e7e048a473)